pytaxize/gnr.py

```diff
@@ -122,6 +123,7 @@ def _gnr_resolve(names='Homo sapiens', source=None, format='json', resolve_once=
                 time.sleep(10)
                 out = requests.get(url=result_url)
                 result_json = out.json()
+            os.remove('__names_list.txt')
 
     data = []
     for each_result in result_json['data']:
```

Comment:
This file can be ignored now, as I already took care of file deletion in https://github.com/sckott/pytaxize/pull/30
Just add the test cases.